lib/init.js

```diff
@@ -166,7 +166,9 @@ module.exports = {
         }
 
         const linter = new (await getPugLint(projectDir))();
+        linter['_basePath'] = projectDir;
```

Comment:
Hmmm, the lint failure is exactly why I checked out the code to see if it would let that happen. Sorry!

I guess this will have to be written as:
```js
// eslint-disable-next-line no-underscore-dangle
linter._basePath = projectDir;
```

And I need to figure out why it wasn't yelling at me locally for that...

Reply:
Can you edit it? I'm just about to leave work for the weekend

Reply:
Forgot I had fixable rules ignored when editing and hadn't saved the modified file. Sorry again!

Reply:
> Can you edit it? I'm just about to leave work for the weekend

Sure.